### ISSUE TITLE
Naming convention not followed by BusinessProfilePerformance (should be MyBusinessPerformance) + API endpoint function missing

### ISSUE THREAD

**Issue by nekromoff** [googleapis/google-api-php-client-services#2237]
#### Environment details

  - OS: Debian
  - PHP version: 8.1.2
  - Package name and version: Release v0.285.0 [Latest](https://github.com/googleapis/google-api-php-client-services/releases/latest)

#### Steps to reproduce

See: https://developers.google.com/my-business/reference/performance/rest
Endpoint is missing from the library.

#### Code example

```php
use Google_Service_MyBusinessPerformance; // potential name respecting similar services
```

**Comment by nekromoff** [comment]
Discovery doc:
https://businessprofileperformance.googleapis.com/$discovery/rest?version=v1

**Comment by nekromoff** [comment]
Also, if you have documented how to add individual discovery docs, I could submit a pull request. I have searched the code, but don't see any repo of the discovery URLs.

**Comment by nekromoff** [comment]
ping

**Comment by nekromoff** [comment]
Does somebody actually take care of these issues? 2 weeks have passed, but this endpoint is still missing. Is this abandoned? Automatic generation does not respect discovery documents as I have mentioned above.

**Comment by unbreak** [comment]
It is called `Google\Service\BusinessProfilePerformance`:

https://github.com/googleapis/google-api-php-client-services/blob/main/src/BusinessProfilePerformance.php

**Comment by nekromoff** [comment]
> It is called `Google\Service\BusinessProfilePerformance`:
> 
> https://github.com/googleapis/google-api-php-client-services/blob/main/src/BusinessProfilePerformance.php

Indeed, this seems to be the case. Why was the naming convention ignored is a mystery.

Further though, there is this endpoint missing from it:
https://developers.google.com/my-business/reference/performance/rest/v1/locations/fetchMultiDailyMetricsTimeSeries

This endpoint was released a few weeks ago. The docs of this github repo say it is updated ever day, however this does not seem to be the case.

**Comment by gellezzz** [comment]
I see new releases but this [endpoint](https://developers.google.com/my-business/reference/performance/rest/v1/locations/fetchMultiDailyMetricsTimeSeries) still missing...

**Comment by unbreak** [comment]
> Indeed, this seems to be the case. Why was the naming convention ignored is a mystery.
I think it's because this service is now named Business Profile, not My Business.

**Comment by ajupazhamayil** [comment]
INFO: The `fetchMultiDailyMetricsTimeSeries` changes have been merged and tracking in this bug: https://github.com/googleapis/google-api-php-client-services/issues/2440.

**Comment by nekromoff** [comment]
> INFO: The `fetchMultiDailyMetricsTimeSeries` changes have been merged and tracking in this bug: #2440.

Great, thanks! I guess I can close this bug, but the naming convention is still an issue to think about.
